Assets/Scripts/PlayerCharacter.cs

```diff
@@ -35,17 +38,25 @@ public class PlayerCharacter : MonoBehaviour
     LayerMask whatIsGround;
     [SerializeField]
     private float groundRadius = 0.2f, speed;
-    private bool facingRight = true, grounded = false, falling=false;
-    private bool isInDeath;
     [SerializeField]
+    [Tooltip("The vector multiplier that slows the character to halt while dying.")]
+    private float deathVector;
```

Comment:
So does it need to be greater than 0 but less than 1? You could use the Range attribute, or at least clarify the tooltip. https://docs.unity3d.com/ScriptReference/RangeAttribute.html

If we can stop the designer from putting in a dumb value, we should.